packages/react-theme-provider/README.md

```diff
@@ -0,0 +1,66 @@
+# @fluentui/react-theme-provider
+
+**React theming component and hook for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+
+# Installation
```

Comment:
nit, these should be h2 (only one h1 per document)

Reply:
done!